lib/settings/widgets/accessibility_profile.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:thunder/shared/snackbar.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+class SettingProfile extends StatelessWidget {
+  final IconData icon;
+  final String name;
+  final String description;
+  final Map<LocalSettings, Object> settingsToChange;
+
+  const SettingProfile({
+    super.key,
+    required this.icon,
+    required this.name,
+    required this.description,
+    required this.settingsToChange,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final ThemeData theme = Theme.of(context);
+    bool recentSuccess = false;
+
+    return ExpandableOption(
+      icon: icon,
+      description: name,
+      child: Column(
+        children: [
+          Text(description),
+          ...settingsToChange.entries.map(
+            (entry) {
+              return Row(
+                children: [
+                  Text('• ${entry.key.label}'),
+                  const Icon(Icons.arrow_right_rounded, size: 20),
+                  Text(_humanizeValue(context, entry.value)),
+                ],
+              );
+            },
+          ),
+          const SizedBox(height: 12),
+          StatefulBuilder(
+            builder: (context, setState) => TextButton(
+              style: ElevatedButton.styleFrom(
+                minimumSize: const Size.fromHeight(45),
+                backgroundColor: theme.colorScheme.primaryContainer.harmonizeWith(theme.colorScheme.errorContainer),
+                disabledBackgroundColor: theme.colorScheme.primaryContainer.harmonizeWith(theme.colorScheme.errorContainer).withOpacity(0.5),
+              ),
+              onPressed: recentSuccess
+                  ? null
+                  : () async {
+                      bool success = true;
+                      final SharedPreferences prefs = (await UserPreferences.instance).sharedPreferences;
+
+                      for (MapEntry<LocalSettings, Object> entry in settingsToChange.entries) {
+                        if (entry.value is bool) {
+                          await prefs.setBool(entry.key.name, entry.value as bool);
+                        } else {
+                          // This should never happen in production, since we should add support for any unsupported types
+                          // before adding a profile containing those types.
+                          success = false;
+                          if (context.mounted) {
+                            showSnackbar(context, AppLocalizations.of(context)!.settingTypeNotSupported(entry.value.runtimeType));
+                          }
+                        }
+                      }
+                      if (context.mounted && success) {
+                        showSnackbar(context, AppLocalizations.of(context)!.profileAppliedSuccessfully(name));
+                        setState(() => recentSuccess = true);
+                        Timer(const Duration(seconds: 5), () async {

```

Comment:
Small suggestion here: I think it might be better to use `Future.delayed` rather than `Timer` here as `Timer` implies that this is a periodic function (even though `Future.delayed` technically uses `Timer` under the hood)!

https://api.flutter.dev/flutter/dart-async/Future/Future.delayed.html

Reply:
Done!